libs/repositories/src/datasources/cms.ts

```
@@ -0,0 +1,34 @@
+import { CmsClient } from '@cowprotocol/cms';
+
+export type CmsClient = ReturnType<typeof CmsClient>;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CmsClient'. Consider to delete it or rename it.

'CmsClient' is defined here:


(lint/suspicious/noRedeclare)